lib/main.rb

```diff
@@ -1,35 +1,50 @@
 require 'json'
 
 class LogParser
-  attr_reader :data
-
-  def initialize file_path = "games.log"
+  def initialize(file_path = "games.log")
     begin
       @file_path = file_path
-      @data = File.readlines(@file_path, chomp: true)
+      @data = File.readlines(file_path, chomp: true)
+      @gamelog_obj_ruby = {
+        @file_path => {
+          lines: line_count(),
+          players: [],
+        } 
+      }
     rescue
-      print "Invalid file path."
+      puts "Invalid file path."
     end
   end
 
   def first_line
     @data.first
   end
+
+  def gamelogJSON
```

Comment:
Por convenção no ruby as funções são declaradas com o formato `snake_case` . 
```suggestion
  def gamelog_json
```

ou mesmo `to_json`

Reply:
Opa, esqueci disso. Tem razão. Já alterei. Valeu!